### ISSUE TITLE
Status fails to process a JSON MOTD containing only a translation

### ISSUE THREAD

**Issue by norhu1130** [py-mine/mcstatus#319]
OSError: Server did not respond with any information!

I am not proceeding with the query.
When progressing status = server.status(), the following error occurs.
```
D:\Lunar>py server.py
Traceback (most recent call last):
  File "server.py", line 4, in <module>
    status = server.status()
  File "C:\Users\norhu1130\AppData\Local\Programs\Python\Python38\lib\site-packages\mcstatus\server.py", line 116, in status
    return self._retry_status(connection, **kwargs)
  File "C:\Users\norhu1130\AppData\Local\Programs\Python\Python38\lib\site-packages\mcstatus\utils.py", line 53, in sync_wrapper
    raise last_exc  # type: ignore # (This won't actually be unbound)
  File "C:\Users\norhu1130\AppData\Local\Programs\Python\Python38\lib\site-packages\mcstatus\utils.py", line 49, in sync_wrapper
    return func(*args, **kwargs)
  File "C:\Users\norhu1130\AppData\Local\Programs\Python\Python38\lib\site-packages\mcstatus\server.py", line 122, in _retry_status
    result = pinger.read_status()
  File "C:\Users\norhu1130\AppData\Local\Programs\Python\Python38\lib\site-packages\mcstatus\pinger.py", line 67, in read_status
    response = self.connection.read_buffer()
  File "C:\Users\norhu1130\AppData\Local\Programs\Python\Python38\lib\site-packages\mcstatus\protocol\connection.py", line 140, in read_buffer
    length = self.read_varint()
  File "C:\Users\norhu1130\AppData\Local\Programs\Python\Python38\lib\site-packages\mcstatus\protocol\connection.py", line 68, in read_varint
    part = self.read(1)[0]
  File "C:\Users\norhu1130\AppData\Local\Programs\Python\Python38\lib\site-packages\mcstatus\protocol\connection.py", line 220, in read
    raise IOError("Server did not respond with any information!")
OSError: Server did not respond with any information!
```

**Comment by kevinkjt2000** [comment]
Looks as if some firewall between the script and the server is interfering. Hard for us to help though given that you haven't shared any versions nor the address of this server. Perhaps the server was still starting or was not running?

**Comment by norhu1130** [comment]
The server is on, and the firewall is fine, too
Server information is available at beeonline.kro.kr

**Comment by kevinkjt2000** [comment]
> the firewall is fine

And it could be the problem. Make sure the necessary UDP port is open in that firewall.

**Comment by norhu1130** [comment]
server.status() Does need a Query?

**Comment by kevinkjt2000** [comment]
I had more time to investigate tonight and found that the port is open, so that's not the issue.

The issue is in the `_parse_description` function not being able to handle `{'translate': 'VDZsaXJnZThoa3RaWlZ1SFdXcStBdz09'}` as input.

**Comment by PerchunPak** [comment]
Looks like `mcsrvstat.us` also can't get server's MOTD:
![image](https://user-images.githubusercontent.com/68118654/175477346-f9b75c7f-7777-4085-beee-c573b5b36583.png)
Could it be server's software problem?

P.S. I also checked MOTD with standard Minecraft, it shows random string.

**Comment by PerchunPak** [comment]
Should be fixed in #335 after [this](https://github.com/py-mine/mcstatus/pull/335/commits/fbb85560e8ae4774cd582f28fc07aac555736be1).

```py
>>> Motd.parse({'translate': 'VDZsaXJnZThoa3RaWlZ1SFdXcStBdz09'}).parsed
[TranslateString(id='VDZsaXJnZThoa3RaWlZ1SFdXcStBdz09'), <Formatting.RESET: 'r'>]
```
